### ISSUE TITLE
Make "Show boards in calendar/tasks" an administrative setting

### ISSUE THREAD

**Issue by nilskamm** [nextcloud/deck#3026]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Is your feature request related to a problem? Please describe.**
Normally boards are shown in calendar/tasks but with very limited functions. So we would like to generally set all boards to NOT be shown in calendar and tasks.

**Describe the solution you'd like**
A setting for administrators to overwrite the Deck setting "Show boards in calendar/tasks" for every user.

**Describe alternatives you've considered**
To make the setting in a config file would be also ok. My only idea to solve this at the moment is to change it in the database table "*_preferences" for every user.

**Comment by juliushaertl** [comment]
Yes, definitely makes sense to have as an option as we already have a per user setting.

**Comment by webermax** [comment]
Would it be possible to link the task list's "close" action with "archiving" a card?

**Comment by stefan-niedermann** [comment]
> Is there a reason why the tasks are greyed out and not editable?

https://help.nextcloud.com/t/deck-updates-from-reminder-app-ios-not-syncing-back/159442/6?u=stefan-niedermann
